aderyn_driver/src/fchelpers.rs

```diff
@@ -0,0 +1,105 @@
+use std::{
+    collections::BTreeMap,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+
+use foundry_compilers::{
```

Comment:
Are we importing the Cyfrin fork?

Reply:
Can you please update the Cyfrin fork ?

Reply:
Done, I've also added you as collaborator on it.